.github/workflows/artifactLinks.yml

```diff
@@ -0,0 +1,20 @@
+name: Artifact links comments creator
+on:
+  workflow_run:
+    workflows: ["CI"]
+    types: [completed]
+
+jobs:
+  artifacts-url-comments:
+    name: Add artifact links to PR and issues
+    runs-on: windows-2022
+    steps:
+      - name: Add artifact links to PR and issues
+        uses: tonyhallett/artifacts-url-comments@v1.1.0
+        env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+            prefix: Do you want to test this code? Here you have an automated build:
+            suffix: WARNING: It may be unstable and result in corrupted configurations or data loss. Use only for testing!
+            format: name
+            addTo: pullandissues
```

Comment:
I think we should only comment on the PR and not issues as well?

Reply:
I think we should use that also on linked issues, is also really useful.